src/admin-events/admin-events.spec.js

```diff
@@ -94,7 +97,14 @@ describe('admin event monitoring', () => {
 
       // Run agent
       const findings = await handleTransaction(txEvent);
-      const alert = [createAlert(eventName, contractName, contractAddress, 'Unknown', 'Low')];
+      const args = {
+        voter: '0x0000000000000000000000000000000000000000',
+        roundId: '0',
+        identifier: '0x0000000000000000000000000000000000000000000000000000000000000000',
+        time: '0',
+        ancillaryData: '0x',
+      };
+      const alert = [createAlert(eventName, contractName, contractAddress, 'Info', 'Low', args)];
```

Comment:
Because the `args` variable is passed to the same function `createAlert` that creates the finding in the handler, we never explicitly check that the data within `args` shows up in the finding metadata.  If we use this `args` variable in the `extractArgs` function, this is what we get:

```javascript
> const strippedArgs = {}
undefined
>       const args = {
...         voter: '0x0000000000000000000000000000000000000000',
...         roundId: '0',
...         identifier: '0x0000000000000000000000000000000000000000000000000000000000000000',
...         time: '0',
...         ancillaryData: '0x',
...       };
undefined
> Object.keys(args).forEach((k) => {
... if (Number.isNaN(k)) { strippedArgs[k] = args[k].toString(); }
... });
undefined
> strippedArgs
{}
```

Reply:
Now that I have the isNaN problem fixed.  If I change anything in args it will throw an error saying it does not match in this test.  Can this be considered 'done'?